networkx/convert_matrix.py

```diff
@@ -229,6 +235,18 @@ def to_pandas_edgelist(
     nodelist : list, optional
        Use only nodes specified in nodelist
 
+    dtype : dtype, default None
+        Use to create the DataFrame. Data type to force.
+        Only a single dtype is allowed. If None, infer.
+
+    order : None
+        An unused parameter mistakenly included in the function.
+        This is deprecated and will be removed in NetworkX v3.0.
```

Comment:
```suggestion
    order : None
        An unused parameter mistakenly included in the function.
        
        .. deprecated:: 2.6
            The is deprecated and will be removed in NetworkX v3.0.
```
The deprecation directive renders nicely in the html docs.